server/views.py

```diff
@@ -110,6 +111,9 @@ def show_all_volunteers(request, page=1):
     if len(search_last_name) != 0 and search_last_name[0] != '':
         q_option |= Q(last_name=search_last_name[0])
 
+    if len(search_id) != 0 and search_id[0].strip():
```

Comment:
```suggestion
    if len(search_id) != 0 and search_id[0] != '':
```

Reply:
this suggestion is just for consistency and will be removed on refactor that will be mentioned on future issue